src/tls.rs

```diff
@@ -679,6 +747,15 @@ extern {
 
     fn SSL_CTX_set_default_verify_paths(ctx: *mut SSL_CTX) -> c_int;
 
+    #[cfg(windows)]
+    fn SSL_CTX_get_cert_store(ctx: *mut SSL_CTX) -> *mut X509_STORE;
+    #[cfg(windows)]
+    fn X509_STORE_add_cert(ctx: *mut X509_STORE, x: *mut X509) -> c_int;
```

Comment:
Nitpick, but the functions in this block are organized by type, so you should move this and the `X509` functions to the end of the block and split them with `// X509_STORE` and `// X509` comments.

Reply:
bloody d2i_, I folded that into ``// X509`` but leem know if you want a separate block